types/react-beautiful-dnd/index.d.ts

```diff
@@ -180,24 +180,24 @@ export interface Scrollable {
 export interface PlaceholderInSubject {
     // might not actually be increased by
     // placeholder if there is no required space
-    increasedBy?: Position | undefined;
+    increasedBy: Position | null | undefined;
```

Comment:
Can it ever be `undefined`? Same for others

Reply:
Copying from external discussion:

The actual values provided by the library are not `undefined` but this matches the internal flow typing of the library.